_capabilities/analysis-showback.md

```diff
@@ -2,7 +2,7 @@
 
 layout: wide
 
-title: Data Analysis and Showback
```

Comment:
@vnm-fodo User wants to change Capability title here. Unsure if this is within scope of what kind of changes you'd like to see. Great to see the starter content elsewhere though!

Reply:
Right on @drumsnob . Let's change this metadata back. Can you also add a few sentences to the description field please based on your content? That would be immensely helpful for our metadata.